From c29169b9d3ef6b2b97ca0b4f1058844de886e31f Mon Sep 17 00:00:00 2001
From: Nath <nathinabob@gmail.com>
Date: Tue, 13 Sep 2022 11:01:45 +0100
Subject: [PATCH] FilterList patch commit

---
 components/FilterList/FilterList.js | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/components/FilterList/FilterList.js b/components/FilterList/FilterList.js
index aa325cb..98d3a49 100644
--- a/components/FilterList/FilterList.js
+++ b/components/FilterList/FilterList.js
@@ -1,6 +1,7 @@
 import React, { useCallback, useEffect, useState } from "react";
 import { SafeAreaView, RefreshControl, View } from "react-native";
-import { FlashList } from "@shopify/flash-list";
+//import { FlashList } from "@shopify/flash-list";
+import { FlatList } from "react-native";
 
 import SearchBar from "./SearchBar";
 import FilterBar from "./FilterBar";
@@ -93,17 +94,21 @@ const FilterList = ({
           <Placeholder placeholder={placeholder} />
         </View>
       ) : (
-        <FlashList
+        <FlatList
           keyExtractor={keyExtractor}
           data={_items}
           renderItem={renderItem}
           refreshControl={
             <RefreshControl refreshing={refreshing} onRefresh={_onRefresh} />
           }
+          style={styles.container}
+          contentContainerStyle={globalStyles.screenGutter}
+          /*
           contentContainerStyle={{
             ...globalStyles.screenGutter,
             ...styles.container,
           }}
+          */
           // Performance settings
           getItemLayout={getItemLayout}
           estimatedItemSize={200}
-- 
2.34.1

